app/views/step_by_step_pages/publish.html.erb

```diff
@@ -16,8 +16,6 @@
 
 <%= render 'shared/steps/step_breadcrumb', links: links %>
 
-<%= render 'shared/steps/form_notice', message: notice, status: "success" %>
```

Comment:
I've noticed that you've taken this out on every page. What was it replaced by? Perhaps it might be better to remove these at the same time as the replacement is added?

Reply:
There is now a generic form notice in the admin template